src/helpers/indexable-helper.php

```diff
@@ -159,4 +206,44 @@ public function dynamic_permalinks_enabled() {
 	public function finish_indexing() {
 		$this->options_helper->set( 'indexables_indexing_completed', true );
 	}
+
+	/**
+	 * Checks whether the indexable has default values in given fields.
+	 *
+	 * @param Indexable $indexable The Yoast indexable that we're checking.
+	 * @param array     $fields    The Yoast indexable fields that we're checking against.
+	 *
+	 * @return bool Whether the indexable has default values.
+	 */
+	public function check_if_default_indexable( $indexable, $fields ) {
+		foreach ( $fields as $field ) {
+			$is_default = $this->check_if_default_field( $indexable, $field );
+			if ( ! $is_default ) {
+				break;
+			}
+		}
+
+		return $is_default;
+	}
+
+	/**
+	 * Checks if an indexable field contains the default value.
+	 *
+	 * @param Indexable $indexable The Yoast indexable that we're checking.
+	 * @param string    $field     The field that we're checking.
+	 *
+	 * @return bool True if default value.
+	 */
+	public function check_if_default_field( $indexable, $field ) {
+		$defaults = $this->default_values;
+		if ( ! isset( $defaults[ $field ] ) ) {
+			return true;
```

Comment:
Would make this `false`. If we don't know of a default it doesn't feel right to say everything is default.

Reply:
Yeah, I put `true` there because I'm sending some fields there that I *don't* want to be checked.

But you're right, I'll put `false` there and won't send those fields for checking :)